dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -23,12 +26,14 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetTimeStamp(winrt::Windows::Foundation::DateTime const& value)
     {
-        
-        //winrt::std::wstring currentLanguage { ApplicationLanguages::Languages().First().Current() };
-        //// DateTimeFormatter dateTimeFormatter{ "longtime" };
-        //winrt::Windows::Foundation::DateTime::
-        //winrt::Windows::Foundation::DateTime { 2022, 1, 13, 16, 25, 30,  }
-        //m_timeStamp = timeStamp;
+        auto seconds{ winrt::clock::to_time_t(value) };
+        struct tm buf;
+        gmtime_s(&buf, &seconds);
+
+        std::wstringstream buffer;
+        buffer << std::put_time(&buf, L"%FT%T");
+
+        m_timeStamp = wil::str_printf<std::wstring>(L" displayTimestamp='%wsZ'", buffer.str().c_str());
```

Comment:
Why the trailing 'Z'? Should above be `std::put_time(..., L"%FT%T%Z")` ?

Reply:
The reason why it's not in std::put_time is because we want the Z to exist as the raw character in the XML to be processed later in AppNotificationBuilder.Show.